client/src/app/app.component.ts

```diff
@@ -71,17 +98,13 @@ export class AppComponent implements OnInit {
   private loadAllProjects() {
     this.projectService
       .getAllProjects()
-      .pipe(
-        catchError(() => {
-          this.isError = true;
-          this.isLoading = false;
-          return EMPTY;
-        })
-      )
+      .pipe(catchError(() => EMPTY))
```

Comment:
maybe I'm wrong, but doesn't the ``EMPTY`` lead to no execution of the function handed over with ``.subscribe(...)``?
I tried it with ``const subscribe = EMPTY.subscribe(() => console.log('Hi'));`` and that didn't say "Hi".
Technically the ``EMPTY`` triggers the complete handler right away, which is not provided here.

Reply:
Good comment! But ... hm ... `EMPTY` docu says `Creates an Observable that emits no items to the Observer and immediately emits a complete notification`. In my case when there's an error then the observable should "stop" and the http interceptor kicks in, that's why the component isn't involved anymore and I found it safe that `catchError` emits "complete". Ok for you to further talk about this outside of this PR as there was no changes introduced to this piece of code?

Reply:
Added gone-lost boolean variables in the `catchError` callback.